liblangutil/Token.h

```diff
@@ -322,6 +324,22 @@ namespace TokenTraits
 			tok == Token::Default || tok == Token::For || tok == Token::Break || tok == Token::Continue || tok == Token::Leave ||
 			tok == Token::TrueLiteral || tok == Token::FalseLiteral || tok == Token::HexStringLiteral || tok == Token::Hex;
 	}
+	constexpr bool isExperimentalSolidityKeyword(Token tok)
+	{
+		return tok == Token::Assembly || tok == Token::Contract || tok == Token::External || tok == Token::Fallback ||
+			tok == Token::Pragma || tok == Token::Import || tok == Token::As || tok == Token::Function || tok == Token::Let ||
+			tok == Token::Return || tok == Token::Type || tok == Token::Bool || tok == Token::If || tok == Token::Else ||
```

Comment:
```suggestion
			tok == Token::Return || tok == Token::Type || tok == Token::If || tok == Token::Else ||
```

I actually imagined to not have any tokens in the beginning - but that would already break the import tests we have at the parsing stage, so probably better to keep the basic structural ones here already - but I'd still remove ``bool`` for now - we may end up making it a special user-defined type (and if not we can re-add it later)

Reply:
I pondered about not having any tokens at all here, but figured that these are at least high probability candidates and fairly basic infrastructure, so considered no harm in letting them stay...apart from `bool` which I agree should not be here since it is a typename.